src/lua/api-gateway/logger/BufferedAsyncLogger.lua

```diff
@@ -373,7 +373,7 @@ function AsyncLogger:flushMetrics()
         local delay = math.random(10, 100)
         local ok, err = ngx.timer.at(delay / 1000, flushMetrics_timerCallback, self)
         if not ok then
-            ngx.log(ngx.WARN, "Could not flushMetrics this time, will retry later. Details: ", err)
+            ngx.log(ngx.DEBUG, "Could not flushMetrics this time, will retry later. Details: ", err)
```

Comment:
Does it happen often not being able to flush metrics? If not, maybe it would be better to leave it on WARN level.

Reply:
Not that often, however since there will be a `retry` I thought I'd move it to `DEBUG` level.